app/scripts/modules/amazon/src/function/configure/FunctionBasicInformation.tsx

```diff
@@ -0,0 +1,271 @@
+    const accountRegions$ = Observable.combineLatest(form.account$, allAccounts$)
+      .switchMap(([currentAccount, _allAccounts]) => AccountService.getRegionsForAccount(currentAccount))
+      .shareReplay(1);
+
+    const allFunctions$ = this.props.app.getDataSource('functions').data$ as Observable<IAmazonFunction[]>;
+    const regionfunctions$ = Observable.combineLatest(allFunctions$, form.account$, form.region$)
+      .map(([allFunctions, currentAccount, currentRegion]) => {
+        return allFunctions
+          .filter(fn => fn.account === currentAccount && fn.region === currentRegion)
+          .map(fn => fn.functionName);
+      })
+      .shareReplay(1);
+
+    accountRegions$
+      .withLatestFrom(form.region$)
+      .takeUntil(this.destroy$)
+      .subscribe(([accountRegions, selectedRegion]) => {
+        // If the selected region doesn't exist in the new list of regions (for a new acct), select the first region.
+        if (!accountRegions.some(x => x.name === selectedRegion)) {
+          this.props.formik.setFieldValue('region', accountRegions[0] && accountRegions[0].name);
+        }
+      });
+
+    Observable.combineLatest(allAccounts$, accountRegions$, regionfunctions$)
+      .takeUntil(this.destroy$)
+      .subscribe(([accounts, regions, existingFunctionNames]) => {
+        return this.setState({ accounts, regions, existingFunctionNames });
+      });
+  }
+
+  private handleRegionChange = (value: string): void => {
+    this.props.formik.setFieldValue('region', value);
+  };
+
+  public render() {
+    const { isNew } = this.props;
+    const { errors, values } = this.props.formik;
+    const { accounts, regions } = this.state;
+    const className = classNames({
+      well: true,
+      'alert-danger': errors.functionName,
+      'alert-info': !errors.functionName,
+    });
+    return (
+      <div className="container-fluid form-horizontal ">
+        <div className="sp-margin-m-bottom">
+          {isNew && (
+            <div className={className}>
+              <strong>Your load balancer will be named: </strong>
+              <HelpField id="aws.function.name" />
+              <span>{this.props.app.name.concat('-').concat(values.functionName)}</span>
+              <Field
+                type="text"
+                style={{ display: 'none' }}
+                className="form-control input-sm no-spel"
+                name="functionName"
+              />
+              {errors.functionName && <ValidationMessage type="error" message={errors.functionName} />}
+            </div>
+          )}
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            fastField={false}
+            name="credentials"
+            label="Account"
+            input={props => (
+              <ReactSelectInput
+                {...props}
+                inputClassName="cloudfoundry-react-select"
+                value={values.credentials}
```

Comment:
do not pass `value` prop to inputs being managed by a `FormikFormField`.  It comes automatically from `{...props}`

Reply:
done. change will appear in the next push.